packages/expo-constants/ios/EXConstants.podspec

```diff
@@ -16,11 +16,17 @@ Pod::Spec.new do |s|
 
   s.dependency 'ExpoModulesCore'
 
+  # Swift/Objective-C compatibility	
+  s.pod_target_xcconfig = {	
+    'DEFINES_MODULE' => 'YES',	
+    'SWIFT_COMPILATION_MODE' => 'wholemodule'	
+  }
```

Comment:
Can you also change the pod name to `ExpoConstants`?

Reply:
Will changing it not require a lot of changes in other modules?

Reply:
Right... it will 😞 Let's leave it as is for now then. I'll take care of updating everything later 😉